<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/about.png</normaloff>icons/about.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QLabel" name="lblHeader">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>501</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Arial Rounded MT Bold'; font-size:36pt; color:#00b0f0;&quot;&gt;Urban Decarb&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblVersionDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>versiondate</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblActiveProject">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:Arial Rounded MT Bold'; font-size:8pt;color:lightblue;&quot;&gt;project &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLogo">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>420</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
